docs/models/advanced/perspective.md

```diff
@@ -0,0 +1,61 @@
+Perspective
```

Comment:
Same story as with `IModel`. The interface lives on as a default noop method in `IBakedModel`. I'll correct the references to the interface, and then could you check this page more thoroughly?